package day42_Exceptions;

public class ThrowsKeyword2 {

    public static void main(String[] args) throws ArithmeticException {

        System.out.println("CobinGroup.Test started");

        System.out.println(8/0);

        System.out.println("CobinGroup.Test completed");


    }

}
